age-calculator/style.css

```diff
@@ -0,0 +1,82 @@
+*,
+*:before,
+*:after{
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+
+}
+body{
+	background-color: #ee6cff;
+}
+.container{
+	width: 40%;
+	min-width: 450px;
+	
+	position: absolute;
+	transform: translate(-50%,-50%);
+	left: 50%;
+	top: 50%;
+	padding : 50px 30px;
+
+}
+.container *{
+	font-family: sans-serif;
+	border:none;
+	outline: none;
+}
+.input{
+	background-color: #080808;
+	padding: 30px 25px;
+	border-radius: 8px;
+	box-shadow: 0 15px 20px rgba(0,0,0,0.3);
+	margin-bottom: 50px;
+
+
+}
+input,button{
+	height: 50px;
+	background-color: #ffffff;
+	color:#080808;
+	font-weight: 500px;
+	border-radius: 5px ;
+
+}
+
+input{
+	width:60%;
+	padding: 0 20px;
+	font-size: 14px;
+}
+button{
+	width: 30%;
+	float: right;
+}
+.output{
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+
+}
+
+.output div{
+	height: 100px;
+	width: 100px;
+	background-color: #080808;
+	border-radius: 5px;
+	color:#fff;
+	display: grid;
+	text-align:center;
+	align-items: center;
+	padding: 20px 0;
+	box-shadow: 0 15px 20px rgba(0,0,0,0.3);
+}
+span{
+	font-size: 30px;
+	font-weight: 500;
+}
+p{
+	font-size: 14px;
+	color: #707070;
+	font-weight: 400;
+}

```

Comment:
Use a CSS beautifier. 
Use this link --  https://codebeautify.org/css-beautify-minify

Reply:
The css is already beautified maam !